development/.env

```diff
@@ -0,0 +1,6 @@
+IP=127.0.0.1
```

Comment:
As a general guideline, we should avoid storing environment variables in our repository, particularly those containing any credentials. I understand the context here is to enable the GitHub actions, but there is the real possibility that a developer might put some sensitive credentials in this file and inadvertently commit it to revision control.

Reply:
@brylie ok i get this but without these information the github worker will not do anything the only solution i can think of is a new docker-compose.yml file like github-compose.yml with some default creds in it.

or let it build without any credentials (if possible)

Reply:
OK, I'm just re-assessing my thinking here. I believe since we are explicitly in a `development/` directory the `.env` file is not so sensitive.